download.html

```diff
@@ -96,32 +96,53 @@
 
   <section id="about-us">
     <div class="container">
-      <div class="center wow fadeInDown">
-        <br>
-      </div>
+      <br>
       <div class="center col-md-12">
-        <p class="lead">
-          Download using the options below.
-        </p>
+        <h3>Install using the options below.
+          <br>
+          Pre-built Rockstor 4 installers are pending Stable status.
+        </h3>
+        <i>Rockstor 4.0.7-0 = Stable Release Candidate 8 (recommended for new installs).</i>
+        <br>
+        <b>Rockstor 4 is now Pi4 & ARM64 Embedded Boot / Server Boot compatible.</b>
       </div>
       <div class="col-sm-offset-2 col-sm-8 col-sm-offset-2">
         <div class="row">
+
           <div class="col-md-6">
             <a href="https://sourceforge.net/projects/rockstor/files" target="_blank">
               <div class="media download-wrap wow fadeInDown">
                 <div class="media-body">
                   <div class="col-sm-8">
-                    <h2 class="media-heading">SourceForge</h2>
+                    <h2 class="media-heading">Rockstor 3 CentOS based</h2>
                     <p>
-                      Files Page for CentOS<br>
-                      based ISO installer
+                      <br>
+                      Near legacy ISO installer (2017) Last Stable v3.9.2-57 (April 2020)
                     </p>
                   </div>
                   <div class="col-sm-4"><img src="/media/images/sf.jpeg" width="70px;" height="70px;"></img></div>
                 </div>
               </div>
             </a>
           </div>
+
+          <div class="col-md-6">
+            <a href="https://github.com/rockstor/rockstor-installer" target="_blank">
+              <div class="media download-wrap wow fadeInDown">
+                <div class="media-body">
+                  <div class="col-sm-8">
+                    <h2 class="media-heading">Rockstor 4 "Built on openSUSE"</h2>
+                    <p>
+                      <br>
+                      Stable Release Candidate v4.0.7 (2021) Build your own installer
+                    </p>
+                  </div>
+                  <div class="col-sm-4"><img src="/media/images/GitHub_Logo.png" width="70px;" height="70px;"></img></div>
```

Comment:
@phillxnet, in Vivaldi (Chrome), the icon looks blurry as it squeezes the image. Using a height of 40px helps a bit (see below); it's a tad smaller but at least it looks sharper. Not very important anyway so it's up to you.
![image](https://user-images.githubusercontent.com/30297881/124918393-f90d9b00-dfc2-11eb-82f5-74219d07f88e.png)

It works well in Firefox with either height setting, though.

Reply:
@FroggyFlox Nice find.
I say we swing back to this one, given issue #1, and move to publish what we have. We will likely have an entirely different setup for these installer items via whatever we find in Hugo/Syna widget wise. Hopefully with a little more sophistication than we currently have access to :).